File: benchmarks/cpp/integer_boolean/primitive_integer/Lift.cpp
#include <iostream>
#include <string>
#include "BUtils.cpp"
#include "BInteger.cpp"

#ifndef Lift_H
#define Lift_H

using namespace std;

class Lift {



    private:



        BInteger floor;

    public:

        Lift() {
            floor = (BInteger(0));
        }

        void inc() {
            floor = floor.plus((BInteger(1)));
        }

        void dec() {
            floor = floor.minus((BInteger(1)));
        }

};
#endif

File: benchmarks/Java_Clojure_Set_Primitives/Lift.java
import de.hhu.stups.btypes.BInteger;
import de.hhu.stups.btypes.BUtils;

public class Lift {





    private BInteger floor;

    public Lift() {
        floor = new BInteger(0);
    }

    public void inc() {
        floor = floor.plus(new BInteger(1));
    }

    public void dec() {
        floor = floor.minus(new BInteger(1));
    }



}
